packages/office-ui-fabric-react/src/components/TextField/TextField.test.tsx

```diff
@@ -299,9 +307,23 @@ describe('TextField with error message', () => {
   it('should render error message when onGetErrorMessage returns a string', () => {
     const validator = jest.fn((value: string) => (value.length > 3 ? errorMessage : ''));
 
+    // function onNotifyResult(errorMessage: string | JSX.Element, value: string | undefined) {
```

Comment:
delete?

Reply:
Oops, thought I pushed a change fixing this. Updated now.